holopy/scattering/tests/test_calculations.py

```diff
@@ -30,65 +31,128 @@
 from holopy.core import detector_grid
 from holopy.core.tests.common import assert_obj_close
 from holopy.scattering.calculations import *
+from holopy.scattering.errors import MissingParameter
 
-scatterer = Sphere(n=1.6, r=.5, center=(5, 5, 5))
-medium_index = 1.33
-locations = detector_grid(shape=(20, 20), spacing=.1)
-wavelen = 0.66
-polarization=(0, 1)
+import xarray as xr
 
-@attr("fast")
-def test_calc_holo():
-    holo = calc_holo(locations, scatterer, medium_index, wavelen, polarization)
+SCATTERER = Sphere(n=1.6, r=.5, center=(5, 5, 5))
+MED_INDEX = 1.33
+LOCATIONS = detector_grid(shape=(20, 20), spacing=.1)
+WAVELEN = 0.66
+POL = (0, 1)
 
 
-def test_calc_field():
-    field = calc_field(locations, scatterer, medium_index, wavelen, polarization)
+class TestCalculations(unittest.TestCase):
+    @attr("fast")
+    def test_calc_holo(self):
+        holo = calc_holo(LOCATIONS, SCATTERER, MED_INDEX, WAVELEN, POL)
+        self.assertTrue(True)
 
+    def test_calc_field(self):
```

Comment:
would be nice to get speed attrs on all of these tests.

Reply:
Addressed in 6aefa22 and 20c0e44